lms/static/scripts/frontend_apps/components/SubmitGradeForm.js

```diff
@@ -69,7 +74,7 @@ const useFetchGrade = student => {
       // Set a flag to cancel the the fetchGrade response from saving to state
       didCancel = true;
```

Comment:
I would rename `didCancel` to `ignoreResults`, because the fetch requests is not actually cancelled.